closed/OpenJ9.gmk

```diff
@@ -280,6 +280,13 @@ else
   FEATURE_SED_SCRIPT += $(call SedDisable,opt_jitserver)
 endif
 
+# Adjust JFR enablement flags.
+ifeq (true,$(OPENJ9_ENABLE_JFR))
+  FEATURE_SED_SCRIPT += $(call SedEnable,opt_jfr)
+else
+  FEATURE_SED_SCRIPT += $(call SedDisable,opt_jfr)
+endif
+
```

Comment:
Please keep the groups in alphabetical order (e.g. JFR should appear before JIT).